src/rgw/rgw-restore-bucket-index

```diff
@@ -118,22 +119,52 @@ get_pool() {
       echo ERROR: unable to determine pool.
       super_exit
   fi
-  echo "$pool"
+  echo "Detected pool: $pool"
```

Comment:
The idea was that this function echoes its result, which is then captured by the caller, likely in a variable. That's why on line 103 we have. But if there's an explicit pool we don't set pool. I think the logic needs to be reworked here.